lib/av_client/types.ts

```diff
@@ -89,6 +89,11 @@ export type BoardItem =
   BallotCryptogramItem |
   CastRequestItem
 
+export type VoterAuthorizationIdentification = {
+  identitiyConfirmationToken?: string;
+  publicKey?: string;
+};
```

Comment:
```suggestion
interface IdentityConfirmationTokenIdentification { identitiyConfirmationToken: string }
interface PublicKeyIdentification { publicKey: string }

export type VoterAuthorizationIdentification = IdentityConfirmationTokenIdentification | PublicKeyIdentification;
```

This will make it so either the identity token or the public key _has_ to be there, rather than both being optional.